0021-plugin-rpc-api/0021-plugin-rpc-api.md

```diff
@@ -0,0 +1,170 @@
+# Plugin RPC API
+
+## Description
+
+> TODO
+
+## Specification
+
+### Send Transfer
+
+#### Request
+
+```http
+POST /rpc/?method=send_transfer&prefix=peer.me HTTP/1.1
+Host: rpchost
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer ABCXYZ
```

Comment:
describe how the bearer token is computed (see https://gist.github.com/michielbdejong/52257fb9e46c62e72fe96020b3afbb62#file-plugin-virtual-s-rest-api-L18-L25)